src/iotEdgeExplorer.ts

```diff
@@ -106,7 +106,8 @@ export class IoTEdgeExplorer extends BaseExplorer {
 
     private async isValidDeploymentJsonSchema(json: object): Promise<boolean> {
         const schema = (await axios.get(Constants.DeploymentJsonSchemaUrl)).data;
-        const ajv = new Ajv({ allErrors: true });
+        const ajv = new Ajv({ allErrors: true, schemaId: "id" });
+        ajv.addMetaSchema(require("ajv/lib/refs/json-schema-draft-04.json"));

```

Comment:
So if draft04 is added here. Can it still parse draft07 version? I am asking this is maybe we will upgrade azure-iot-edge-deployment-2.0 to draft07 later.

Reply:
Yes, it is supported, we do not need to update any code when our schema change to draft07 later